### PR TITLE
Conference tests

### PR DESCRIPTION
**Pull request by natebrennand**
0% -> 79%

everything that's lacking is because we don't have a slick way to do HTTP testing yet

### CODE REVIEW

File: conference/callback.go [hunk 1]
```diff
@@ -12,7 +12,6 @@ type Callback struct {
 
 // Parse the form encoded callback into a Callback struct
 func (cb *Callback) Parse(req *http.Request) error {
-
```
Comment (chelpu) [inline conference/callback.go:15]: :'(

File: conference/conference_test.go [hunk 2]
```diff
@@ -50,3 +53,27 @@ func TestParticipantBuild(t *testing.T) {
...
+	}
+	qs := f.getQueryString()
+
+	assert.Contains(t, qs, "Status=completed")
+	assert.Contains(t, qs, "FriendlyName=board+meeting")
+}
+
+func TestConferenceGetBadReq(t *testing.T) {
+	var c Conference
+	respC, err := testAccount.Get("sldfkj")
```
Comment (chelpu) [inline conference/conference_test.go:72]: #effort

File: conference/participant.go [hunk 3]
```diff
@@ -107,12 +107,12 @@ type ParticipantList struct {
...
 
 // ListParticipants queries for a list of participants in the conference with confSid.
-func (act Account) ListParticipants(u ParticipantUpdate, confSid string) (ParticipantList, error) {
+func (act Account) ListParticipants(confSid string) (ParticipantList, error) {
 	var pl ParticipantList
 	if !validateConferenceSid(confSid) {
 		return pl, errors.New("Invalid conference sid")
 	}
-	err := common.SendGetRequest(fmt.Sprintf(participant.List, act.AccountSid, confSid)+u.getParticipantQueryString(), act, &pl)
+	err := common.SendGetRequest(fmt.Sprintf(participant.List, act.AccountSid, confSid), act, &pl)
```
Comment (chelpu) [inline conference/participant.go]: if we remove u, how will someone filter the results of the list?

File: conference/conference_test.go (hunk 2 above)
Comment (natebrennand) [inline conference/conference_test.go:72]: it's an invalid SID, that's all that mattered to tipsy me on the train

File: conference/participant.go (hunk 3 above)
Comment (natebrennand) [inline conference/participant.go]: you were passing in a `ParticipantUpdate` struct, that's used for a different call. Also there is no filtering on participants in a conference

Comment (chelpu) [inline conference/participant.go]: Yeah there is (https://www.twilio.com/docs/api/rest/participant#list-get-filters) and it just happens to be the same as a `Participant Update` with the member `Muted`

Comment (natebrennand) [inline conference/participant.go]: oops, my bad. well we still need to rename that struct to something else

Comment (natebrennand) [inline conference/participant.go]: maybe `ParticipantStatus`?

Comment (chelpu) [inline conference/participant.go]: Fine with me, do you want me to add it or do you want to?

Comment (natebrennand) [inline conference/participant.go]: either works for me

Comment (chelpu) [inline conference/participant.go]: I'll do it
P.S. LOVE the branch name "conferernce_tests"

Comment (natebrennand) [inline conference/participant.go]: i don't have autocompletion on the CLI. and was tipsy on a train

**Comment by chelpu** [comment]
this is a :+1: for me, but it would be good if you could review what I added @natebrennand

**Comment by natebrennand** [comment]
:100:

**Pull request merged**